### ISSUE TITLE
duplicate functionality

### ISSUE THREAD

**Issue by DanGrayson** [UniMath/UniMath#1029]
```
prod_precategory = 
λ C D : precategory, prod_precategory_data C D,, prod_precategory_is_precategory C D
     : precategory → precategory → precategory

precategory_binproduct = 
λ C D : precategory,
precategory_binproduct_data C D,, is_precategory_precategory_binproduct_data C D
     : precategory → precategory → precategory
```

**Comment by m-lindgren** [comment]
This can be closed. Fixed by #1462.

**Comment by benediktahrens** [comment]
@m-lindgren : thanks a lot! Closing.
